cosmos-gui/src/app.js

```diff
@@ -49,6 +51,10 @@ var scEndpoint = config.clusters.storage.endpoint;
 var ccPrivKey = config.clusters.computing.private_key;
 var ccUser = config.clusters.computing.user;
 var ccEndpoint = config.clusters.computing.endpoint;
+var httpsOptions = {
+    key: fs.readFileSync(config.private_ley_filekey),
```

Comment:
I think there's a typo here: `private_ley_filekey` -> `private_key_file` :) Apart from this, I may have asked this before :p but have you confirmed that this works if `config.private_ley_filekey` and `certificate_file` are blank (`''`) as in the default configuration? :)